root/etc/s6-overlay/s6-rc.d/svc-pydio-cells/run

```diff
@@ -1,18 +1,11 @@
 #!/usr/bin/with-contenv bash
 
-_term() {
```

Comment:
Does v4 shutdown cleanly without the need for a finish script now?

Reply:
I ran out of time for testing. This should have been a draft PR. Oops

Reply:
Previous version used to stop all processes upon receiving a SIGINT, but v4 doesn't. It often leaves half of the forked processes running. Now the finish script takes care of them all.